includes/definitions/mrv-optidriver.yaml

```diff
@@ -0,0 +1,10 @@
+os: mrv-optidriver
+group: mrv
```

Comment:
@laf I based that off of the pre-existing `./includes/definitions/mrvld.yaml`, which denotes the `mrv` group. MRV has a larger suite of devices than just OptiDriver (OptiPacket, OptiSwitch), so I would guess that they could have definition OS types.